src/scripts/main.js

```diff
@@ -1,3 +1,206 @@
 'use strict';
+// Variables
 
-// write your code here
+const startBtn = document.querySelector('.start');
+const score = document.querySelector('.game-score');
+const startMsg = document.querySelector('.message-start');
+const winMsg = document.querySelector('.message-win');
+const loseMsg = document.querySelector('.message-lose');
+
+// Field Variables
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)
Fix all cases